### ISSUE TITLE
Fe FoldSearchEnd() removes color highlighting

### ISSUE THREAD

**Issue by John-Appleseed** [embear/vim-foldsearch#3]
**Issue**
I'm compressing the search results with shift+return however foldsearch removes color when `:Fe` is executed after `:Fs`
### SearchFoldToggle Test

nnoremap <S-return> :call SearchFoldToggle()<cr>

```
let g:search_fold_toggle = 1
function! SearchFoldToggle()
    if g:search_fold_toggle
        Fs
        let g:search_fold_toggle = 0
    else
        Fe
        let g:search_fold_toggle = 1
    endif
endfunction
```

What is a way to restore color?

**Comment by Konfekt** [comment]
I found that :Fe disables syntax hightlighting as well. My work around was
```
nnoremap Fe :noautocmd Fe<CR>:syntax on<CR>zv
```
Hope that helps in the meanwhile.

**Comment by embear** [comment]
Should be resolved in current version. Please reopen if it still persists.

**Comment by embear** [comment]
Hopefully fixed the problem in commit 22f8c0fb388f. Did not happen on Linux but only on Windows.
